src/components/LanguageButton/LanguageButton.js

```diff
@@ -1,13 +1,18 @@
 import React from "react";
 import { Button } from "react-native-elements";
 import PropTypes from "prop-types";
+import AsyncStorage from "@react-native-community/async-storage";
 import { styles } from "./styles";
+import { changeLocale } from "../../i18n";
 
 const LanguageButton = props => {
-  const { title, langKey, navigation } = props;
-  /* eslint-disable-next-line */
-  const onPress = key => {
-    navigation.navigate("auth");
+  const { title, langKey, navigation, screenProps, navigationRoute } = props;
+
+  const onPress = async key => {
+    changeLocale(key);
```

Comment:
probably would be better to put this line inside `onLocaleChange ` callback, so component would know only about singe callback from props, not external dependencies such as AsyncStorage and changeLocale

Reply:
fixed it